src/backend/aspen/api/schemas/samples.py

```diff
@@ -61,6 +61,11 @@ def get(self, key: Any, default: Any = None) -> Any:
                 if self._obj.show_private_identifier
                 else None
             ),
+            "collection_location": (
+                self._obj.location
+                if self._obj.location != "" and self._obj.location != "NaN"
```

Comment:
will it be these strings or an actual null object? Should we also add a check for that or use .get() to avoid key errors?

Reply:
I think this refers to the `location` string field in the DB, which is `not null` ... though I'm definitely a little worried that we have `NaN` values in that field 😨

Reply:
Yeah as Jess said it's a non-nullable field.

Reply:
I have no idea how `NaN` values got in there but they all seem to be CZB samples from earlier this year.